third_party/move/move-compiler-v2/tests/variable-coalescing/cant_coalesce_1.opt.exp

```diff
@@ -17,6 +17,14 @@ public fun m::test($t0: u64): u64 {
   2: return $t1
 }
 
+
+Diagnostics:
+warning: Unused assignment to `x`. Consider removing or prefixing with an underscore: `_x`
+  ┌─ tests/variable-coalescing/cant_coalesce_1.move:3:17
+  │
+3 │         let x = a + a;
+  │                 ^^^^^
```

Comment:
It would really be nice if we could point to `x` here instead of `a + a`.  Is that really not possible without bytecode changes?

Reply:
Tracked in https://github.com/aptos-labs/aptos-core/issues/13841